IO/IODispatcher.cpp

```diff
@@ -324,7 +321,86 @@ void TrajectoriesJPSV04::WriteFooter()
  * FLAT format implementation
  */
 
-TrajectoriesFLAT::TrajectoriesFLAT() : Trajectories() {}
+TrajectoriesFLAT::TrajectoriesFLAT() : Trajectories()
+{
+    // Add header, info and output for speed
+    _optionalOutputHeader[OptionalOutput::speed] = "V\t";
+    _optionalOutputInfo[OptionalOutput::speed]   = "#V: speed of the pedestrian (in m/s)";
+    _optionalOutput[OptionalOutput::speed]       = [](const Pedestrian * ped) {
+        std::stringstream stream;
+        stream << std::setprecision(2) << std::fixed << ped->GetV().Norm() << "\t";
+        return stream.str();
```

Comment:
Maybe you could use libfmt here as well? like: 
```
#include "general/Format.h"
...
return fmt::format(check:fmt("{:.2f}\t"), ped->GetV().Norm());
```

Reply:
Should I change it for every option?